src/pages/auth/signup.tsx

```diff
@@ -168,6 +168,24 @@ const Page: NextPage = () => {
             <FormErrorMessage>{errors.password?.message}</FormErrorMessage>
           </FormControl>
 
+          <FormControl isRequired isInvalid={}>
```

Comment:
@michaelgrigoryan25 What is the code for checking if password input is equal to confirm password input? I wasn't able to figure that out.

Reply:
```ts
!isNil(errors.field_name) // where field_name is one of the fields specified in the zod schema.
```

Documentation of [`_.isNil()`](https://lodash.com/docs/#isNil)

Reply:
With commit e65d846, I got it to stop showing the password box errors on the confirmPassword box, but I can't get it to show my custom confirm password error, can you help me out with this? Sorry, I have no experience with Zod

Reply:
No worries @import-brain. Since Can you just change the code of the `<FormError>` field of the `confirmPassword`'s `<FormControl>`->`<FormErrorMessage>`  from this

```html
<FormErrorMessage>{errors.password?.message}</FormErrorMessage>
````

to this

```html
<FormErrorMessage>{errors.confirmPassword?.message}</FormErrorMessage>
```